src/CIQS-CLI-Extension/azext_ciqs/custom.py

```diff
@@ -105,10 +113,7 @@ def getDeploymentParameters(cmd, deploymentId, subscription=None):
     subscription[optional]: Provides an alternate subscripton to use if desired.
     """
     currentProvisioningStep = viewCurrentProvisioningStep(cmd, deploymentId, subscription=subscription)
-    try:
-        parameters = currentProvisioningStep.parameters
-    except KeyError:
-        return '[]'
```

Comment:
The try was no longer needed. This was left over from before the SDK refactor when the model was a dictionary. Now that there is a class model for the provisioning steps, it will always have parameters and a KeyError should not be possible there.